app/components/copyable-code.ts

```diff
@@ -30,7 +28,7 @@ export default class CopyableCodeComponent extends Component<Signature> {
       () => {
         this.codeWasCopiedRecently = false;
       },
-      1000,
+      config.x.copyConfirmationTimeout,
```

Comment:
Hmm, not sure about using a single value for this - even though we're okay with a (small) fixed value for tests, for production, it's likely that in different contexts this'd need to be different. 

For example, if this was a secret link / password, we'd choose to just keep the confirmation timeout alive for longer (to force the user to take the next action, and not distract) 

Maybe we could just do something like `env === 'test' ? 10 : 1000` here?

Reply:
I thought that according to testing "conventions" the code tested should not be aware of the fact that it's being tested, meaning that `env === "tested"` is considered a very un-orthodox to do?

How about adding a second configuration variable, `copyConfirmationTimeoutImportantMessage: 10000`, for cases where you'd need a longer confirmation? I don't think this would need a TON of variations... maybe short/medium/long at maximum.

Reply:
On a second thought, `env === 'test' ? 10 : 1000` is not as evil as 

```js
if (config.environment !== 'test') {
  navigator.clipboard.writeText(this.args.code);
}
```

because the former just changes the timeout, while the latter skips an API invocation completely 😱

Reply:
But the counter-argument is that there already like 8 places where this timeout is hard-coded in various places, and each of them would need to include the env check, and I'm sure someone will forget the env check when he adds a ninth place :P 

I thought about adding a `static confirmationTimeout` fields to the components themselves, but they are a PITA to override in tests. Much easier to have them in config.

Reply:
So, your call, @rohitpaulk ! :D 

Centralize them into config vars or not?

Reply:
> because the former just changes the timeout, while the latter skips an API invocation completely

Yep - agree with this, the latter is indeed error prone. 

@VasylMarchuk I'd just use `env === 'test' ? 10 : 1000` where necessary, doesn't seem like that'd cause too many issues

Reply:
Perfect, I don't really have a strong opinion on this, just wanted to have some else's to confirm, feel perfectly fine with `env === 'test' ? 10 : 10000`